apps/web/pages/booking/[uid].tsx

```diff
@@ -443,8 +443,10 @@ export default function Success(props: SuccessProps) {
                           )}
                           {bookingInfo?.attendees.map((attendee) => (
                             <div key={attendee.name + attendee.email} className="mb-3 last:mb-0">
-                              {attendee.name && <p>{attendee.name}</p>}
-                              <p data-testid={`attendee-${attendee.email}`}>{attendee.email}</p>
+                              {attendee.name && (
+                                <p data-testid={`attendee-name-${attendee.name}`}>{attendee.name}</p>
+                              )}
+                              <p data-testid={`attendee-email-${attendee.email}`}>{attendee.email}</p>
```

Comment:
We needed to update test that were using attendee-${value}, so they keep working.
@G3root